src/MQTT433gateway.cpp

```diff
@@ -236,6 +239,19 @@ void setup() {
     }
   });
 
+  settings.registerChangeHandler(SYSLOG, [](const Settings &s) {
+    if (syslogLog) {
+      Logger.removeHandler(*syslogLog);
+      delete syslogLog;
+    }
+    if (s.syslogLevel.length() != 0 && s.syslogHost.length() != 0 &&
+        s.syslogPort != 0) {
+      syslogLog = new SyslogLogTarget();
+      Logger.addHandler(Logger.stringToLevel(s.syslogLevel), *syslogLog);
+      syslogLog->begin(s.deviceName, s.syslogHost, s.syslogPort);
```

Comment:
`begin()` should be before registering the handler.

Reply:
In principle yes. As long as we don't have real concurrency it doesn't matter.

Reply:
please change